source/plugin/offline-userplayer-data.rst

```diff
@@ -0,0 +1,58 @@
+===================
+Offline Player Data
+===================
+
+It may be necessary for plugins to access player data even when the player is offline.
+One could guess that ``Sponge.getServer().getPlayer()`` returning a ``Player`` can be used for this.
```

Comment:
I presume "one could guess" means the same as "you might think"? The latter seems clearer to me.
Also, if you change "has to" into "must" it makes it clearer that it is imperative.